network/dnscrypt-proxy/rc.dnscrypt-proxy

```diff
@@ -1,134 +1,79 @@
-#!/bin/bash
-
-CONFIGFILE="/etc/default/dnscrypt-proxy"
-DAEMON="/usr/sbin/dnscrypt-proxy"
-
-. $CONFIGFILE
-
-start_instance() {
-    if [ ! -r ${DNSCRYPTCONFIG[$1]} ]; then
-        echo "No configuration for instance $1 found!"
-        return
-    fi
-    if [ -z ${PIDFILE[$1]} ]; then
-        echo "No PID configuration for instance $1 found!"
-        return
-    fi
-    if [ -z ${USER[$1]} ]; then
-        echo "No user configuration for instance $1 found!"
-        return
-    fi
-    if [ -r ${PIDFILE[$1]} ]; then
-        echo "dnscrypt-proxy (instance $1) already running!"
-        return
-    fi
-
-    mkdir -p $(dirname ${PIDFILE[$1]})
-    # The child (unprivileged) process needs write access or the PID will not
-    # be written.
-    chmod 0700 $(dirname ${PIDFILE[$1]})
-    chown ${USER[$1]} $(dirname ${PIDFILE[$1]})
-
-    # The new Go-based dnscrypt-proxy no longer has the ability to daemonize.
-    # In the absence of a standard Slackware daemon tool we'll use nohup. :(
-    nohup $DAEMON -config ${DNSCRYPTCONFIG[$1]} -pidfile ${PIDFILE[$1]} >> /dev/null 2>&1 &
-}
+#!/bin/sh
 
-stop_instance() {
-    if [ ! -r ${DNSCRYPTCONFIG[$1]} ]; then
-        echo "No configuration for instance $1 found!"
-        return
-    fi
-    if [ -z ${PIDFILE[$1]} ]; then
-        echo "No PID configuration for instance $1 found!"
-        return
-    fi
-    if [ ! -r ${PIDFILE[$1]} ]; then
-        echo "dnscrypt-proxy (instance $1) is not running!"
-        return
-    fi
-    echo "Stopping dnscrypt-proxy (instance $1)..."
-    kill $(cat ${PIDFILE[$1]})
-}
+# Init file for dnscrypt-proxy
 
-status_instance() {
-    if [ ! -r ${DNSCRYPTCONFIG[$1]} ]; then
-        echo "No configuration for instance $1 found!"
-        return
-    fi
-    if [ -z ${PIDFILE[$1]} ]; then
-        echo "No PID configuration for instance $1 found!"
-        return
-    fi
-    if [ ! -r ${PIDFILE[$1]} ]; then
-        echo "dnscrypt-proxy (instance $1) is not running."
-        return
-    fi
-    PID=$(cat ${PIDFILE[$1]})
-    if [ -z "$PID" ]; then
-        echo "PID file is empty! dnscrypt-proxy (instance $1) does not appear to be running, but there is a stale PID file."
-    elif kill -0 $PID ; then
-        echo "dnscrypt-proxy (instance $1) is running."
-    else
-        echo "dnscrypt-proxy (instance $1) is not running, but there is a stale PID file."
-    fi
-}
+PRGNAM=dnscrypt-proxy
```

Comment:
This variable seems unused?

Reply:
I will be cleaning the rc script up. I forgot to check the sanity of that script before pushing.